app/code/Magento/Checkout/CustomerData/Cart.php

```diff
@@ -90,8 +91,13 @@ public function getSectionData()
     {
         $totals = $this->getQuote()->getTotals();
         $subtotalAmount = $totals['subtotal']->getValue();
+        $summaryCount = $this->getSummaryCount();
+        if ($summaryCount>0) {
+            $store = ObjectManager::getInstance()->get(\Magento\Framework\Locale\Resolver::class);
+            $summaryCount = \Zend_Locale_Format::toNumber($this->getSummaryCount(), ['locale' => $store->getLocale()]);
```

Comment:
Please use the `$summaryCount` variable instead of `$this->getSummaryCount()`, since it's allready defined.

Reply:
I think, using the `Zend_Locale_Format` directly is not the best option here.
I would recommend checking for possible utilities in Magento Framework. e.g. in `Magento\Framework\Filter` namespace or other already available solutions.